src/index.js

```diff
@@ -27,18 +28,40 @@ Sentry.init({
     return event;
   },
 });
-checkCookie();
+function FallbackComponent({ _error, _componentStack, resetError }) {
+  const { t } = useTranslation();
+  return (
+    <div className="App-body">
+      <h2>{t("Oops!")}</h2>
+      <p>{t("Something, somewhere went terribly wrong.")}</p>
+      <p>{t("Please reload the page.")}</p>
+      <button type="button" onClick={resetError}>
+        {t("Reload")}
+      </button>
+    </div>
+  );
+}
 
-ReactDOM.render(
+const container = document.getElementById("root");
+const root = createRoot(container);
+
+root.render(
   <Provider store={store}>
-    <Suspense fallback={null}>
-      <App />
-    </Suspense>
-  </Provider>,
-  document.getElementById("root")
+    <PersistGate loading={null} persistor={persistor}>
+      <Suspense fallback={null}>
+        <React.StrictMode>
+          <Sentry.ErrorBoundary fallback={FallbackComponent} showDialog>
+            <BrowserRouter>
+              <App />
+            </BrowserRouter>
+          </Sentry.ErrorBoundary>
+        </React.StrictMode>
+      </Suspense>
+    </PersistGate>
+  </Provider>
 );
 
 // If you want your app to work offline and load faster, you can change
 // unregister() to register() below. Note this comes with some pitfalls.
 // Learn more about service workers: https://bit.ly/CRA-PWA
-serviceWorker.unregister();
+// serviceWorker.unregister();
```

Comment:
Comment out or remove? Should this be `register`?

Reply:
There is no support for a serviceWorker. Commenting out this line did not introduce a serviceWorker